README.md

```diff
@@ -1,2 +1,18 @@
-# baips
-BitAssets Improvement Proposals
+# BitAssets Improvement Proposal
+
+BAIP stands for BitAssets Improvement Proposal but can also seen as an
+improvement *consensus*. BitAssets are BitShares Committee's SmartCoins
+including but not limited to BitCNY, BitUSD and BitBTC. A BAIP is a design document providing information to the
+BitShares community, or describing a new feature for BitAssets or change its parameters
+or price feed formula, logic and source. The BAIP should provide a concise technical specification of the
+feature or the idea and a rationale for it. It may not only describe technical
+improvements but also document *best-practises* and recommendations.
+
+The distinct document [BAIP Purpose and Guidelines](baip-0001.md) gives a more
+detailed explanation.
+
+# Available BSIPs
+
+Number             | Title                                                    | Owner             | Type           | Status
```

Comment:
I think it's better to update `Owner` to `Author`

Reply:
Change to Author